src/ipv4.rs

```diff
@@ -41,6 +41,14 @@ pub trait Address:
     ///
     /// in lieu of implementing Into<[u8; 4]>, this allows this crate to easily get at the
     /// underlying byte array.
+    ///
+    /// # Example
+    /// ```
+    /// use addrs::ipv4;
```

Comment:
Some of this is distracting. I think I want to clean it up with hidden use statements.